src/integrations/langfuse.ts

```diff
@@ -1,17 +1,18 @@
 const langfuseParams = {
-    publicKey: process.env.LANGFUSE_PUBLIC_KEY,
-    secretKey: process.env.LANGFUSE_SECRET_KEY,
-    baseUrl: process.env.LANGFUSE_HOST,
-}
+  publicKey: process.env.LANGFUSE_PUBLIC_KEY,
+  secretKey: process.env.LANGFUSE_SECRET_KEY,
+  baseUrl: process.env.LANGFUSE_HOST,
+};
 
 import { Langfuse } from 'langfuse';
 
 const langfuse = new Langfuse(langfuseParams);
 
-export async function getPrompt(
-    id: string,
-    version?: number,
-): Promise<string> {
-    const prompt = await langfuse.getPrompt(id, version);
-    return prompt.getLangchainPrompt();
+export async function getPrompt(id: string, version?: number): Promise<string> {
+  const prompt = await langfuse.getPrompt(id, version);
+  const outputPrompt = prompt.getLangchainPrompt();
+  if (typeof outputPrompt !== 'string') {
+    return JSON.stringify(outputPrompt);
+  }
```

Comment:
This is the gist of the change, we want to stringify the result if it's not a `string` type.